reports/weekly-cancelled-court-moves/02-weekly-cancelled-court-moves-cronjob.yaml

```diff
@@ -0,0 +1,102 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: weekly-cancelled-court-moves-report
+spec:
+  schedule: "30 6 * * 1"
+  concurrencyPolicy: "Forbid"
+  successfulJobsHistoryLimit: 5
+  failedJobsHistoryLimit: 3
+  jobTemplate:
+    spec:
+      ttlSecondsAfterFinished: 345600
+      backoffLimit: 0
+      activeDeadlineSeconds: 7200
+      template:
+        spec:
+          serviceAccountName: "book-a-secure-move-api"
+          containers:
+            - name: weekly-cancelled-court-moves
+              image: "ghcr.io/ministryofjustice/hmpps-devops-tools:latest"
+              command: ["bash", "/scripts/run-report.sh"]
+              volumeMounts:
+                - name: report-scripts
+                  mountPath: /scripts
+                  readOnly: true
+                - name: report-sql
+                  mountPath: /report
+                  readOnly: false
+              env:
+                - name: DB_INSTANCE
+                  valueFrom:
+                    secretKeyRef:
+                      name: read-rds-instance-hmpps-book-secure-move-api-preprod
```

Comment:
Is preprod correct here, or should it be prod?

Reply:
preprod is fine for the moment as we're manually applying anyway - Helm will handle namesapces when you pick that up with Josh :)